src/main/java/dk/bec/polonez/reservationsystem/controller/FeatureController.java

```diff
@@ -25,12 +26,24 @@ public FeatureController(FeatureService featureService){
 
     @GetMapping("{id}")
     public Feature getFeatureById(@PathVariable long id) {
```

Comment:
**warn** You shouldn't expose JPA objects (like ```Feature```) to consumers of your REST API. You should use some dedicated Dto for your REST layer (```FeatureDto``` or ```ResponseFeatureDto```).